doc_source/calico.md

```diff
@@ -4,12 +4,34 @@
 
 **To install Calico on your Amazon EKS cluster**
 
-1. Apply the Calico manifest from the [`aws/amazon-vpc-cni-k8s` GitHub project](https://github.com/aws/amazon-vpc-cni-k8s)\. This manifest creates daemon sets in the `kube-system` namespace\.
+1. Apply the Calico manifest from the [`aws/amazon-vpc-cni-k8s` GitHub project](https://github.com/aws/amazon-vpc-cni-k8s)\.
+   This manifest creates the calico-node `DaemonSet` in the `kube-system` namespace\.
 
    ```
-   kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/v1.0.0/config/v1.0/aws-k8s-cni-calico.yaml
+   kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/v1.0.1/config/v1.0/calico.yaml
    ```
 
+**To upgrade Calico on your Amazon `v1.0.0` EKS cluster**
```

Comment:
I would guess that you do not need the backticks around v1.0.0. Also, I am wondering, will this procedure work to: a) install Calico on an EKS v1.0.0 cluster AND b) upgrade Calico on an EKS v1.0.0 cluster. Or is there a third procedure to handle a or b.

Reply:
Agreed re: backticks.

In terms of upgrading the AWS CNI plugin from v1.0.0 to a v1.0.1 - this is a question for Eric @nrdlngr: Is it correct to assume that the AWS CNI plugin has already been updated before people get to this page, or should we include instructions for updating that here as well as updating Calico?